### DISCUSSION TITLE
added mailaddress already in use

### DISCUSSION THREAD

**Discussion by ElsDeSw** [community/community#119963] [New to GitHub]
on adding a new mailadress, I get reply that this mailaddress is already in use. What does it mean. Is it possible to have two github accounts?

**Comment by ABDE-LKADER** [comment]
Yes, it's possible to have multiple GitHub accounts using the same email address, but each account must have a unique username. If you're getting a message that an email address is already in use, it means it's associated with an existing GitHub account.
